Datasources: Add toggle to control default behaviour of 'Manage alerts via Alerts UI' toggle (#98441)
grafana/grafana: docs/sources/setup-grafana/configure-grafana/_index.md

<hr />

## [datasources]

### default_manage_alerts_ui_toggle

> Default behaviour for the "Manage alerts via Alerting UI" toggle when configuring a Datasource. Only works in case there's no configured value beforehand controlled by the datasources' `jsonData.manageAlerts` prop.

Reviewer: ```suggestion
Default behavior for the "Manage alerts via Alerting UI" toggle when configuring a data source. It only works if the data source's `jsonData.manageAlerts` prop does not contain a previously configured value.
```
